auth/tests/driver/axiosAuth.driver.ts

```diff
@@ -198,4 +198,27 @@ export class AxiosAuthDriver implements AuthDriver {
         access_token: data.access_token,
       }));
   }
+
+  async revokeToken(refreshToken: string): Promise<RevokeTokenResponse> {
```

Comment:
May be worth adding this to the interface `auth.driver.ts` now that we can log people out 🚀

Reply:
Good shout, will do